chainerx_cc/chainerx/routines/math.h

```diff
@@ -10,6 +10,86 @@
 
 namespace chainerx {
 
+class AddOp : public Op {
+public:
+    static const char* name() { return "Add"; }
+
+    virtual void Call(const Array& x1, const Array& x2, const Array& out) = 0;
+};
+
+Array Add(const Array& x1, const Array& x2);
```

Comment:
How about leaving function declarations in the original locations for better browsability?
Ops and wrapper functions do not have to be 1-to-1 correspondence.

Reply:
Fixed.